### DISCUSSION TITLE
Login With Facebook response always get undefined

### DISCUSSION THREAD

**Discussion by jykoaster** [nuxt-community/auth-module#1469] [Q&A]
version:5.0.0-1624817847.21691f1

Why I always get undefined after login response?

nuxt.config.js

           auth: {
              strategies: {
                facebook: {
                  endpoints: {
                    userInfo:
                      'https://graph.facebook.com/v6.0/me?fields=id,first_name,last_name,email',
                  },
                  clientId: '**',
                  scope: ['public_profile', 'email'],
                },
              },
              redirect: {
                login: '/',
                logout: '/',
                callback: '/',
                home: '/',
              },
            },

And here is my code

      await context.$auth
        .loginWith('facebook')
        .then((response) => {
          console.log(response)  // undefined
        })
        .catch((err) => {
          console.log(err)
        })

**Comment by devzom** [comment]
The same happend to me, can really use the Facebook schema, that's why had to implement the RAW FB logic > https://developers.facebook.com/docs/facebook-login/web/#

**Comment by RonAlmog** [comment]
Same thing here. not sure how to tackle this.

**Comment by toniengelhardt** [comment]
I had to set `responseType: 'code'` in the facebook strategy options, but I'm routing auth through the backend via Django allauth, not sure if that helps.
